src/main/java/org/jabref/gui/maintable/MainTableNameFormatter.java

```diff
@@ -19,6 +19,31 @@
         abbrAuthorNames = preferences.getBoolean(JabRefPreferences.ABBR_AUTHOR_NAMES);
     }
 
+    /**
+     * Format a name field for the table, according to user preferences and with latex expressions translated if
+     * possible.
+     *
+     * @param nameToFormat The contents of the name field.
+     * @return The formatted name field.
+     */
+    public String formatNameLatexFree(final String nameToFormat) {
```

Comment:
The old `formatName` method is still used?

Reply:
I don't think so.
I have been avoiding removing functionality or methods. As I am very junior, from a Java experience point of view, I was hoping that it would make reviewing and merging my code easier. If it has the opposite effect, I'll try to be more aggressive in adding, removing, and reformatting :P

Reply:
Thanks, that's a good approach and indeed makes reviewing easier. However, if you a method is no longer used you can safely remove it.